lib/ueberauth/client/oauth2.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Ueberauth.Client.OAuth2 do
+  def get_token!(_, params \\ [], _ \\ [], _ \\ []) do
+    [code: code] = params
+
+    case code  do
+      "abc123" -> %{
```

Comment:
These are the actual two codes returned by Slack?

Reply:
Nope. These are just mock codes that are sent to Slack for them to authenticate.